Make docs more LLM friendly (#29566)
dagster-io/dagster: docs/docs/dagster-plus/features/authentication-and-access-control/rbac/user-roles-permissions.md

---

> description: Implement role-based access control (RBAC) in Dagster+ to manage user

Reviewer: ```suggestion
description: Role-based access control (RBAC) enables you to grant specific permissions to users in your Dagster+ organization, ensuring that users only have access to what they need.
```

>   roles and permissions, ensuring precise access levels from Admin to Viewer across

Reviewer: ```suggestion
```

>   deployments.

Reviewer: ```suggestion
```
